bin/build

```diff
@@ -10,6 +10,8 @@ printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build
   "$CIRCLE_BUILD_URL" \
 > version.json
 
+cp .env-dist .env
```

Comment:
This should probably be `[ ! -f .env ] && cp .env-dist .env`. Additionally I don't think this needs to be part of the build step. ATMO has it as an instruction in the README (not ideal because it requires another manual step) and also in `bin/test` but as I understand it this file is meant to be used by docker-compose and not as an artifact in the container. We should exclude it from being copied to the container if we want to keep it in the build script.